### ISSUE TITLE
[Feature Request] Allow differentiation of login from WEB vs. IRC

### ISSUE THREAD

**Issue by teward** [znc/znc#935]
Adminlog and current functions currently identify failed logins by IP, but it does not state whether it's IRC based or webadmin based.  This can be confusing for those of us who put stuff in front of the web interface to point to a different IP.

Is it possible to somehow add into the system (for adminlog and failed login attempt data when connected to ZNC) a way to say whether the failed login was from web or irc?

**Comment by dgw** [comment]
This is good. :+1:

Depending on what kind of proxy or what-have-you you have in front of the webui, the latest versions of ZNC (since 1.6, I believe) understand `X-Forwarded-For` headers. If the adminlog module or some other part of ZNC doesn't use the IP from that header when present, that's a bug.

**Comment by teward** [comment]
This could also be useful if there's nothing in front either, such as someone using webadmin vs. irc, etc. whether `X-Forwarded-For` exists or not.

**Comment by dgw** [comment]
Of course, yes. Was just trying to be helpful in case you weren't even getting the actual client IP. This feature request gets a :+1: from me no matter what.
